Sources/OpenFeature/EventHandler.swift

```diff
@@ -2,29 +2,24 @@ import Combine
 import Foundation
 
 public class EventHandler: EventSender, EventPublisher {
-    private let eventState: CurrentValueSubject<ProviderEvent, Never>
+    private let lastSentEvent = PassthroughSubject<ProviderEvent?, Never>()
 
-    convenience init() {
-        self.init(.notReady)
```

Comment:
The EventHandler doesn't hold a half-baked "ProviderState" anymore, state is moved to OpenFeatureAPI